### PR TITLE
docs: add steps for running the proxy via Node

### PR DESCRIPTION
**Pull request by thomasheartman**
Also: restructure the steps relating to running it via docker to align
better and to avoid repeating information.

### CODE REVIEW

**Comment by thomasheartman** [comment]
[How do you run the Unleash proxy locally without docker?](https://trello.com/c/KPOowYoM/682-how-do-you-run-the-unleash-proxy-locally-without-docker)

File: website/docs/sdks/unleash-proxy.md [hunk 1]
```diff
@@ -41,46 +41,73 @@ Regardless of how you choose to run the it, the proxy will need access to these
 
   When using an environment variable to set the proxy secrets, the value should be a comma-separated list of strings, such as `secret-one,secret-two`.
 
-For more information, check out the [documentation on GitHub](https://github.com/Unleash/unleash-proxy).
+There are many more configuration options available. You find all [available options on github](https://github.com/Unleash/unleash-proxy#available-options).
```
Comment (sighphyre) [inline website/docs/sdks/unleash-proxy.md:44]:
```suggestion
There are many more configuration options available. You'll find all [available options on github](https://github.com/Unleash/unleash-proxy#available-options).
```

File: website/docs/sdks/unleash-proxy.md [hunk 2]
```diff
@@ -41,46 +41,73 @@ Regardless of how you choose to run the it, the proxy will need access to these
...
 
 ### Running the proxy via Docker
 
 The easiest way to run Unleash is via Docker. We have published a [docker image on docker hub](https://hub.docker.com/r/unleashorg/unleash-proxy).
 
-**Step 1: Pull**
+1. **Pull the Proxy image**
+   ```bash
+   docker pull unleashorg/unleash-proxy
+   ```
```
Comment (sighphyre) [inline website/docs/sdks/unleash-proxy.md:54]: Just out of interest, do we need the pull step? A "run" should pull behind the scenes if there's no local image right?

File: website/docs/sdks/unleash-proxy.md [hunk 3]
```diff
@@ -41,46 +41,73 @@ Regardless of how you choose to run the it, the proxy will need access to these
...
+   app.listen(port, () =>
+       console.log(`Unleash Proxy listening on http://localhost:${port}/proxy`),
+   );
+   ```
 
-**Step 3: verify**
+### Verify that the proxy is working
 
-In order to verify the proxy you can use curl and see that you get a few evaluated feature toggles back:
+In order to verify the proxy you can use curl and see that you get a few evaluated feature toggles back. Assuming that the proxy is running at port 3000 and that your proxy client key is `some-secret`, you could run this command :
```
Comment (sighphyre) [inline website/docs/sdks/unleash-proxy.md:102]:
```suggestion
In order to verify the proxy you can use curl and see that you get a few evaluated feature toggles back. Assuming that the proxy is running on port 3000 and that your proxy client key is `some-secret`, you could run this command :
```

**Review by sighphyre — APPROVED** [review]
Love it! Few small things, nothing blocking

File: website/docs/sdks/unleash-proxy.md (hunk 3 above)
Comment (sighphyre) [reply website/docs/sdks/unleash-proxy.md:102]: "On" is idiomatic to me but I'm open to being told that I'm wrong

File: website/docs/sdks/unleash-proxy.md (hunk 2 above)
Comment (chriswk) [reply website/docs/sdks/unleash-proxy.md:54]: It's the recommended way, it'll only pull if there's no local image unless the user runs with `--always-pull`, so for making sure that they run with the latest image, remembering to pull is a good idea :)

Comment (sighphyre) [reply website/docs/sdks/unleash-proxy.md:54]: Fair enough! Apparently I'm terribly lazy when it comes to docker and I should stop being lazy

Comment (thomasheartman) [reply website/docs/sdks/unleash-proxy.md:54]: Cool! I didn't know that either. I guess the option is to add the `--always-pull` flag as we've done a couple other places?

**Pull request merged**